jupyterhub_singleuser_profiles/ui/src/CustomElements/VarForm.js

```diff
@@ -39,22 +41,28 @@ class VarForm extends React.Component {
     }
 
     enterVariable(event) {
-        this.setState({key: event.target.text}, () => {this.checkKey(this.state.key)})
+        this.setState({key: event.target.text})
+        this.setState({type: "password"})
+        document.getElementById(this.state.key + "check"+this.props.formIndex).checked = true
```

Comment:
It would be good to have a function which will generate the `id` - this way you have the "check" string hardcoded all over the place